src/main/java/com/laserfiche/repository/api/clients/impl/model/FuzzyType.java

```diff
@@ -2,6 +2,8 @@
 
 import com.fasterxml.jackson.annotation.JsonCreator;
 import com.fasterxml.jackson.annotation.JsonValue;
+import com.fasterxml.jackson.annotation.JsonInclude;
```

Comment:
Again, we could batch run "optimize imports" to remove unused imports. Ideally, we could have the ability to resolve dependencies during code generation, but I expect it will be tough (the code gen probably will need to have knowledge of the entire project to do the resolution correctly, I am not sure).

Reply:
Thanks for mentioning the "optimize imports" in intellij. I wasn't familiar with that. I used it to remove unnecessary imports.